src/inlineExpression.tsx

```diff
@@ -171,8 +171,8 @@ export function InlineRenderer({ value }: { value?: string }): JSX.Element {
   }
 
   // Explicitly render text/plain
-  const preferredMimeType = rendermime.preferredMimeType(mimeBundle ?? {});
-  if (preferredMimeType === 'text/plain') {
+  const preferred = rendermime.preferredMimeType(mimeBundle ?? {}, 'any');
```

Comment:
`, 'any'` is the only change (made the variable name shorter so it stayed on one line...)

I **think** that this is the correct choice:

> How to consider safe/unsafe factories. If 'ensure', it will only consider safe factories. If 'any', any factory will be considered. If 'prefer', unsafe factories will be considered, but only after the safe options have been exhausted.

By choosing `any`, we consider all render types, which I think that we only want the plain text renderer to be chosen if no other option is available.

Worst case is that in some weird trusted mode mismatch, we would get quotes around text in that case.